packages/hdwallet-walletconnect-bridge/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@shapeshiftoss/hdwallet-walletconnect-bridge",
+  "version": "1.27.0",
```

Comment:
Let's bump the minor version for this feature. Run `yarn lerna:version-minor` and a version bump for all packages + commit will be performed automatically.